scwx-qt/source/scwx/qt/map/radar_product_layer.cpp

```diff
@@ -172,9 +170,10 @@ void RadarProductLayer::UpdateSweep()
                               std::try_to_lock);
    if (!sweepLock.owns_lock())
    {
-      logger_->debug("Sweep locked, deferring update");
+      // logger_->debug("Sweep locked, deferring update");
```

Comment:
Recommend `trace` instead of commenting. This one has been useful for livelock detection in the past.